creativecloud/features/interactive-components/slider-tray/slider-tray.js

```diff
@@ -199,6 +247,52 @@ function uploadImage(media, layer) {
   });
 }
 
+function continueToPs(layer, imgObj) {
+  layer.querySelectorAll('.continueButton').forEach((btn) => {
+    btn.addEventListener('click', async () => {
+      const actionJSONData = [
+        {
+          _obj: 'make',
+          _target: [{ _ref: 'adjustmentLayer' }],
+          using: {
+            _obj: 'adjustmentLayer',
+            type: {
+              _obj: 'hueSaturation',
+              adjustment: [
+                {
+                  _obj: 'hueSatAdjustmentV2',
+                  hue: Math.round(imgObj.hue || 0),
+                  lightness: 0,
+                  saturation: Math.round(imgObj.saturation || 0),
+                },
+              ],
+              colorize: false,
+              presetKind: {
+                _enum: 'presetKindType',
+                _value: 'presetKindCustom',
+              },
+            },
+          },
+        },
+      ];
+      const psurls = [
+        'https://photoshop.adobe.com',
+        'https://dev.photoshop.adobe.com',
+        'https://pr.photoshop.adobe.com/?PR=48898',
+        'https://localhost.corp.adobe.com:3000',
+      ];
+      const { openInPsWeb } = await import('../../../deps/openInPsWeb/openInPsWeb.js');
+      const imageData = await (await fetch(imgObj.imgSrc)).blob();
+      openInPsWeb(
+        psurls[1],
```

Comment:
Should be picked form PS env url value and not hard coded

Reply:
Please review it again